File: Server/SuperBoard/src/main/java/com/narara/superboard/boardmember/interfaces/dto/log/AddBoardMemberInfo.java
package com.narara.superboard.boardmember.interfaces.dto.log;

import com.narara.superboard.board.document.BoardInfo;

public record AddBoardMemberInfo(
        Long memberId,
        String memberNickname,
        Long boardId,
        String boardName
) implements BoardInfo { }

File: Android/core/model/src/main/java/com/ssafy/model/activity/boardmember/AddBoardMemberInfo.kt
package com.ssafy.model.activity.boardmember

data class AddBoardMemberInfo(
    val memberId: Long,
    val memberNickname: String,
    val boardId: Long,
    val boardName: String,
){
    override fun toString(): String {
        return memberNickname
    }
}
